lms/djangoapps/grades/signals/handlers.py

```diff
@@ -191,7 +191,8 @@ def enqueue_subsection_update(sender, **kwargs):  # pylint: disable=unused-argum
             event_transaction_id=unicode(get_event_transaction_id()),
             event_transaction_type=unicode(get_event_transaction_type()),
             score_db_table=kwargs['score_db_table'],
-        )
+        ),
+        countdown=2,
```

Comment:
This is the mechanism for [TNL-6424](https://openedx.atlassian.net/browse/TNL-6424). I've chosen a default delay of 2 seconds, which is roughly double the worst 99th-percentile response time for `X/XBlock/Handler/CapaDescriptorWithMixins.xmodule_handler/problem_check` as reported by NewRelic for the past 7 days: https://rpm.newrelic.com/accounts/88178/applications/3343327/transactions?tw%5Bend%5D=1485791351&tw%5Bstart%5D=1485186551#id=5b225765625472616e73616374696f6e2f507974686f6e2f58426c6f636b2f48616e646c65722f4361706144657363726970746f72576974684d6978696e732e786d6f64756c655f68616e646c65722f70726f626c656d5f636865636b222c22225d